content/applications/sales/sales/invoicing/invoicing_policy.rst

```diff
@@ -4,78 +4,81 @@ Invoice based on delivered or ordered quantities
+In Odoo *Sales*, the basic sales flow starts with the creation of a quotation. Then, that quotation
+is sent a customer. Next, it needs to be confirmed, which turns the quotation into a sales order.
+This, in turn, creates an invoice.
+
+The following is a breakdown of how invoicing policy rules impact the aforementioned sales flow:
 
-- **Invoice what is ordered**: No impact on this basic sales flow. Indeed, you can invoice as soon
-  as the sale is confirmed.
+- :guilabel:`Invoice what is ordered`: No impact on the basic sales flow. An invoice is created as
+  soon as a sale is confirmed.
+- :guilabel:`Invoice what is delivered`: Minor impact on sales flow, because the delivered quantity
+  needs to be manually entered on the sales order. Or, the *Inventory* app can be installed and used
+  to confirm the delivered quantity before creating an invoice with the *Sales* app.
 
-- **Invoice what is delivered**: Small impact on sales flow because you will have to manually
-  enter the delivered quantity on the sales order or to install the **Inventory App** to confirm the
-  delivered quantity before creating an invoice, with the **Sales App**. Indeed, if you try to
-  create an invoice without validating the delivered quantity, you will receive an error message as
-  below.
+.. warning::
+   If a user attempts to create an invoice without validating the delivered quantity, the following
+   error message appears: :guilabel:`There is no invoiceable line. If a product has a Delivered
+   quantities invoicing policy, please make sure that a quantity has been delivered.`.
 
-  .. image:: invoicing_policy/invoicing_policy_3.png
-     :class: img-thumbnail
-     :alt: How the choice of your invoicing policy impacts your sales flow on Odoo Sales?
+  .. image:: invoicing_policy/invoicing-policy-error-message.png
+     :align: center
+     :alt: If Delivered Quantities invoicing policy is chosen, ensure a quantity has been delivered.
 
 .. note::
-   Once the quotation is confirmed and that the status went from **Quotation sent** to
-   **Sales order**, you are able to see your delivered and invoiced quantities directly from your
-   sales order (it is true for both rules).
+   Once a quotation is confirmed, and the status changes from :guilabel:`Quotation sent` to
+   :guilabel:`Sales order`, the delivered and invoiced quantities are available to view, directly
+   from the sales order. This is true for both invoicing policy rule options.
 
-   .. image:: invoicing_policy/invoicing_policy_4.png
+   .. image:: invoicing_policy/invoicing-policy-order-lines.png
       :align: center
-      :class: img-thumbnail
-      :alt: How to see your delivered and invoiced quantities on Odoo Sales?
+      :alt: How to see your delivered and invoiced quantities on Odoo Sales.
 
-   Odoo will automatically add the quantities to the invoice (even if it is a partial delivery).
+   Odoo automatically adds the quantities to the invoice, even if it's a partial delivery.
```

Comment:
It's unclear which "quantities" this sentence is referring to. The delivered or invoiced quantities? When does this happen?